dashboard/app/controllers/home_controller.rb

```diff
@@ -41,8 +41,8 @@ def index
     if current_user
       # We skip the redirect if the user does not have an age set, because we
       # pop up the age interstitial on /home
-      if current_user.student? && current_user.primary_script && current_user.age.present?
-        redirect_to script_next_path(current_user.primary_script)
+      if current_user.student? && current_user.primary_script
+        redirect_to script_path(current_user.primary_script)
```

Comment:
We'll have to test this with yesterday's problem scenario for new google classroom students. @ewjordan has the details and knows how to set up a repro. It might be worth still skipping the redirect when age is missing (unless of course we can confirm it works with this new path). If we do remove that check, also remove the associated comment above ;)

Reply:
The repro is pretty easy, from the dashboard-console you can do the same thing the test does:

```
student.birthday = nil
student.age = nil
student.save(validate: false)
```

and then just navigate to the root (not home) and make sure that you can enter an age on whatever page you land on.